Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -41,15 +41,19 @@ function GetCommentsFromIssues
     Write-Verbose "Transforming SonarQube analysis issues to PR comments"
     
     $comments = New-Object System.Collections.ArrayList
-
     foreach ($issue in $issues)
     {
         $priority = GetCommentPriority $issue
 
-        Assert ( ![String]::IsNullOrWhiteSpace($issue.message) ) "Internal error: the SonarQube reported issues do not have a property named 'message'"
-        Assert ( ![String]::IsNullOrWhiteSpace($issue.line) ) "Internal error: the SonarQube reported issues do not have a property named 'line'"        
-        Assert ( ![String]::IsNullOrWhiteSpace($issue.relativePath) ) "Internal error: the SonarQube reported issues do not have a property named 'relativePath'"
-        Assert ( ![String]::IsNullOrWhiteSpace($issue.rule) ) "Internal error: the SonarQube reported issues do not have a property named 'rule'"
+        if ([String]::IsNullOrWhiteSpace($issue.line) -or $issue.line -lt 0)
+        {
+            Write-Verbose "A SonarQube issue - $($issue.message) from $($issue.relativePath) - has no line associated with it. Placing it at the beginning of the file."
+            $issue.line = 0
+        }
+        
+        Assert ( ![String]::IsNullOrWhiteSpace($issue.message) ) ("Internal error: a SonarQube issue do not have a property named 'message' " + (DumpObject($issue)))
+        Assert ( ![String]::IsNullOrWhiteSpace($issue.relativePath) ) ("Internal error: the SonarQube reported issues do not have a property named 'relativePath' " +(DumpObject($issue)))
+        Assert ( ![String]::IsNullOrWhiteSpace($issue.rule) ) ("Internal error: the SonarQube reported issues do not have a property named 'rule' " + (DumpObject($issue)))
```

Comment:
Minor: typos/errors in all of the Assert messages:
* first message: "a SonarQube issue _does not have_..."
* second and third messages: both refer to "reported issues" when they are only talking about a single issue. Should be in the same form as the first message.